mongoose.h

```diff
@@ -833,6 +833,24 @@ struct timeval {
 #define MG_ENABLE_PROFILE 0
 #endif
 
+#ifndef MG_ENABLE_TCPIP_DRIVER_INIT    // mg_mgr_init() will also initialize
+#define MG_ENABLE_TCPIP_DRIVER_INIT 0  // enabled built-in driver for
```

Comment:
I really would prefer that to be 1, and avoid setting it in the mongoose_custom.h